### PR TITLE
Add partnercourse related commands

### PR DESCRIPTION
**Pull request by dlathyun**
Add PartnerCourse list and delete commands

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/SeplendidParser.java [hunk 1]
```diff
@@ -132,11 +141,24 @@ private LocalCourseCommand getLocalCourseCommandWithoutArg(String userInput, Str
...
     }
 
-    private PartnerCourseAddCommand getPartnerCourseCommand(String userInput, String actionWord, String arguments)
-            throws ParseException {
+    private PartnerCourseCommand getPartnerCourseCommandWithArg(
+            String userInput, String actionWord, String arguments) throws ParseException {
         switch(actionWord) {
         case PartnerCourseAddCommand.ACTION_WORD:
             return new PartnerCourseAddCommandParser().parse(arguments);
+        case PartnerCourseDeleteCommand.ACTION_WORD:
```
Comment (lamchenghou) [inline src/main/java/seedu/address/logic/parser/SeplendidParser.java:149]: Nice.

File: src/test/java/seedu/address/logic/commands/PartnerCourseDeleteCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.commands;
```
Comment (lamchenghou) [inline src/test/java/seedu/address/logic/commands/PartnerCourseDeleteCommandTest.java:1]: After this review, maybe make rest of tests in separate PR?

**Review by lamchenghou — APPROVED** [review]
Looks good, other than `PartnerCourseDeleteCommand#execute`. 

I've added to `SeplendidModelManager` api which might make deletion simpler, can you help [check it out](https://github.com/AY2324S1-CS2103T-W10-2/tp/pull/97)?

File: src/test/java/seedu/address/logic/commands/PartnerCourseDeleteCommandTest.java (hunk 2 above)
Comment (dlathyun) [reply src/test/java/seedu/address/logic/commands/PartnerCourseDeleteCommandTest.java:1]: ok!
